lib/just_audio_media_kit.dart

```diff
@@ -69,14 +68,14 @@ class JustAudioMediaKit extends JustAudioPlatform {
     /// The name of the library is generally `libmpv.so` on GNU/Linux and `libmpv-2.dll` on Windows.
     String? libmpv,
   }) {
-    if ((UniversalPlatform.isLinux && linux) ||
+    if (!((UniversalPlatform.isLinux && linux) ||
         (UniversalPlatform.isWindows && windows) ||
         (UniversalPlatform.isAndroid && android) ||
         (UniversalPlatform.isIOS && iOS) ||
-        (UniversalPlatform.isMacOS && macOS)) {
-      registerWith();
-      MediaKit.ensureInitialized(libmpv: libmpv);
-    }
+        (UniversalPlatform.isMacOS && macOS))) return;
```

Comment:
This shouldn't be done. It should be the user's choice whether to enable `just_audio_media_kit` to be used instead of the default plugin on macOS.

Reply:
I'm a bit confused about this one... I've *cleaned* my code, so brackets won't confuse you:
```dart
if (!(
  (UniversalPlatform.isLinux && linux) ||
  (UniversalPlatform.isWindows && windows) ||
  (UniversalPlatform.isAndroid && android) ||
  (UniversalPlatform.isIOS && iOS) ||
  (UniversalPlatform.isMacOS && macOS)
)) return;
```
I haven't removed the `isMacOS` check, it is still present in my code. Or am I missing something here?

---

Basically, my code does the exact same thing as your did, but instead of this:
```dart
if (condition) {
  do_something();
}
```
I've changed it to this:
```dart
if (!condition) return;

do_something();
```

Reply:
Oh, I'm sorry, I didn't read it properly.
Though readability-wise, I liked it more my way, it's more straightforward since it's just one "if" statement, and a pretty long one, too.